File: NotificationSystem/NotificationSystem.Data/Classes/clsReports.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Diagnostics;
using System.Globalization;
using System.IO;
using System.Reflection;
using System.Runtime.CompilerServices;
using System.Security;
using System.Threading.Tasks;
using Microsoft.VisualBasic;
using System.Data;
using NotificationSystem.NotificationSystem.Data.xsReportsTableAdapters;
using static NotificationSystem.NotificationSystem.Data.xsReports;

namespace NotificationSystem.NotificationSystem.Data.Classes
{
	class clsReports
	{


		public DataTable GetAgReport()
		{
			try
			{
				AgentTableAdapter adpAgents = new AgentTableAdapter();
				AgentDataTable tblAgent = adpAgents.GetData();

				return tblAgent;
			}
			catch (Exception)
			{
				throw;
			}
		}
		public DataTable GetAgentByAgentID(int AgentID)
		{
			try
			{
				AgentTableAdapter adpAgents = new AgentTableAdapter();
				AgentDataTable tblAgent = adpAgents.GetDataByAgentID(AgentID);

				return tblAgent;
			}
			catch (Exception)
			{
				throw;
			}
		}


		public DataTable GetCustReport()
		{
			try
			{
				CustomerTableAdapter adpCustomers = new CustomerTableAdapter();
				CustomerDataTable tblCustomer = adpCustomers.GetData();

				return tblCustomer;
			}
			catch (Exception)
			{
				throw;
			}
		}

		public DataTable GetCustomerByCustomerID(int CustomerID)
		{
			try
			{
				CustomerTableAdapter adpCustomers = new CustomerTableAdapter();
				CustomerDataTable tblCustomer = adpCustomers.GetDataByCustomerID(CustomerID);

				return tblCustomer;
			}
			catch (Exception)
			{
				throw;
			}
		}
	}
}

File: NotificationSystem/NotificationSystem.Data/Classes/clsReports.vb
Imports System
Imports System.Data

Namespace NotificationSystem.NotificationSystem.Data.Classes
    Friend Class clsReports


        Public Function GetAgReport() As DataTable
            Try
                Dim adpAgents As NotificationSystem.NotificationSystem.Data.xsReportsTableAdapters.AgentTableAdapter = New NotificationSystem.NotificationSystem.Data.xsReportsTableAdapters.AgentTableAdapter()
                Dim tblAgent As NotificationSystem.NotificationSystem.Data.xsReports.AgentDataTable = adpAgents.GetData()

                Return tblAgent
            Catch __unusedException1__ As Exception
                Throw
            End Try
        End Function
        Public Function GetAgentByAgentID(AgentID As Integer) As DataTable
            Try
                Dim adpAgents As NotificationSystem.NotificationSystem.Data.xsReportsTableAdapters.AgentTableAdapter = New NotificationSystem.NotificationSystem.Data.xsReportsTableAdapters.AgentTableAdapter()
                Dim tblAgent As NotificationSystem.NotificationSystem.Data.xsReports.AgentDataTable = adpAgents.GetDataByAgentID(AgentID)

                Return tblAgent
            Catch __unusedException1__ As Exception
                Throw
            End Try
        End Function


        Public Function GetCustReport() As DataTable
            Try
                Dim adpCustomers As NotificationSystem.NotificationSystem.Data.xsReportsTableAdapters.CustomerTableAdapter = New NotificationSystem.NotificationSystem.Data.xsReportsTableAdapters.CustomerTableAdapter()
                Dim tblCustomer As NotificationSystem.NotificationSystem.Data.xsReports.CustomerDataTable = adpCustomers.GetData()

                Return tblCustomer
            Catch __unusedException1__ As Exception
                Throw
            End Try
        End Function

        Public Function GetCustomerByCustomerID(CustomerID As Integer) As DataTable
            Try
                Dim adpCustomers As NotificationSystem.NotificationSystem.Data.xsReportsTableAdapters.CustomerTableAdapter = New NotificationSystem.NotificationSystem.Data.xsReportsTableAdapters.CustomerTableAdapter()
                Dim tblCustomer As NotificationSystem.NotificationSystem.Data.xsReports.CustomerDataTable = adpCustomers.GetDataByCustomerID(CustomerID)

                Return tblCustomer
            Catch __unusedException1__ As Exception
                Throw
            End Try
        End Function
    End Class
End Namespace
